### PR TITLE
Add Les Mes Ajudes to landing page

### PR DESCRIPTION
**Pull request by maukoquiroga**
Capture d’écran 2019-11-21 à 17 51 50

### CODE REVIEW

**Comment by sandcha** [comment]
Thank you! It's great to add Mes Ajudes! 
I would keep both Barcelona and France examples as it also says that an example could be applied to multiple countries 🤔 .

File: content/_index.en.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 title: "Turn law into software"
 usecases:
-  mes_aides:
+  les_meves_ajudes:
     title: Calculate citizens' rights
     description: From an individual situation, calculate any variable of a national tax and benefit system.
     caption_1: uses OpenFisca and its
-    caption_2: to inform French citizens on their eligibility to national and local benefits.
-  embauche: 
+    caption_2: to inform Barcelonian citizens on their eligibility to social benefits.
```
Comment (jvalduvieco) [inline content/_index.en.md]: ``
```suggestion
    caption_2: to inform Barcelona citizens on their eligibility to social benefits.
```

File: layouts/index.html [hunk 2]
```diff
@@ -29,13 +29,13 @@ <h1 id="products">{{ $.Param "products" }}  🚀</h1>
...
-				<p>{{ $.Param "usecases.mes_aides.description" }}</p>
+				<h3>{{ $.Param "usecases.les_meves_ajudes.title" }}</h3>
+				<p>{{ $.Param "usecases.les_meves_ajudes.description" }}</p>
 			</div>
 
 			<figure>
-				<video class="screenshot" src="/videos/mes-aides.mp4" poster="/img/mes-aides.png" controls></video>
-				<figcaption><a href="https://beta.gouv.fr/startup/mes-aides">Mes Aides</a> {{ $.Param "usecases.mes_aides.caption_1" }} <a href="http://openfisca.org/doc/openfisca-web-api/index.html">Web API</a> {{ $.Param "usecases.mes_aides.caption_2" }}</figcaption>
+				<img class="screenshot" src="/img/les-meves-ajudes.png" alt="Les Mes Ajudes">
+				<figcaption><a href="https://ajuntament.barcelona.cat/lesmevesajudes/">Les Mes Ajudes</a> {{ $.Param "usecases.les_meves_ajudes.caption_1" }} <a href="https://openfisca.org/doc/openfisca-web-api/index.html">Web API</a> {{ $.Param "usecases.les_meves_ajudes.caption_2" }}</figcaption>
```
Comment (jvalduvieco) [inline layouts/index.html]:
```suggestion
				<figcaption><a href="https://ajuntament.barcelona.cat/lesmevesajudes/">Les meves ajudes</a> {{ $.Param "usecases.les_meves_ajudes.caption_1" }} <a href="https://openfisca.org/doc/openfisca-web-api/index.html">Web API</a> {{ $.Param "usecases.les_meves_ajudes.caption_2" }}</figcaption>
```

**Comment by maukoquiroga** [comment]
Capture d’écran 2019-11-23 à 10 05 37

**Comment by maukoquiroga** [comment]
Capture d’écran 2019-11-23 à 10 12 22

**Pull request merged**